### PR TITLE
feat: add sso (enterprise login) docs under platform

### PR DESCRIPTION
**Pull request by hf**
Adds the SSO docs for enabling SSO with Google Workspace (GSuite), Azure AD and Okta to the Platform section.

### CODE REVIEW

File: apps/docs/pages/guides/platform/sso/sso-with-azure.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  id: 'sso-with-azure',
+  title: 'Setup SSO with Azure AD',
+  description: 'Configuration instructions for single sign-on with Azure AD (Entra)',
+}
+
+Supabase supports single sign-on (SSO) using Microsoft Azure AD.
+
+## Step-by-step guide
+
+### Step 1: Add and register an Enterprise Application
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-azure.mdx:14]: Adding custom anchors to make the URL a bit cleaner, easier to link to, and maintain in the future.
```suggestion
## Step 1: Add and register an Enterprise Application [#add-and-register-enterprise-application]
```

File: apps/docs/pages/guides/platform/sso/sso-with-azure.mdx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+Open up the [Azure Active
+Directory](https://portal.azure.com/#view/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/~/Overview)
+dashboard for your Azure account.
+
+Click the *Add* button then *Enterprise application*.
+
+![Azure AD console: Default Directory Overview](/docs/img/sso-azure-step-01.png)
+
+### Step 2: Choose Create your own application
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-azure.mdx:24]:
```suggestion
## Step 2: Choose Create your own application [#create-application]
```

File: apps/docs/pages/guides/platform/sso/sso-with-azure.mdx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+You'll be using the custom enterprise application setup for Supabase.
+
+![Azure AD console: Browse Azure AD Gallery, select: Create your own
+application](/docs/img/sso-azure-step-02.png)
+
+### Step 3: Fill in application details
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-azure.mdx:31]:
```suggestion
## Step 3: Fill in application details [#add-application-details] 
```

File: apps/docs/pages/guides/platform/sso/sso-with-azure.mdx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+In the modal titled *Create your own application* enter the name you wish
+Supabase to be available to your Azure AD users. `Supabase` works in most
+cases.
+
+Make sure to choose the third option: *Integrate any other application you
+don't find in the gallery (Non-gallery)*.
+
+![Azure AD console: Create your own application modal](/docs/img/sso-azure-step-03.png)
+
+### Step 4: Choose the Set up single sign-on option
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-azure.mdx:42]:
```suggestion
## Step 4: Choose the Set up single sign-on option [#set-up-single-sign-on]
```

File: apps/docs/pages/guides/platform/sso/sso-with-azure.mdx [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+Before you get to assigning users and groups, which would allow accounts in
+Azure AD to access Supabase, you need to configure the SAML details that allows
+Supabase to accept sign in requests from Azure AD.
+
+![Azure AD console: Supabase custom enterprise application, selected Set up
+single sign-on](/docs/img/sso-azure-step-04.png)
+
+### Step 5: Select SAML single sign-on method
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-azure.mdx:51]:
```suggestion
## Step 5: Select SAML single sign-on method [#saml-sso]
```

File: apps/docs/pages/guides/platform/sso/sso-with-azure.mdx [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+Supabase only supports the SAML 2.0 protocol for Single Sign-On, which is an
+industry standard.
+
+![Azure AD console: Supabase application, Single sign-on configuration screen,
+selected SAML](/docs/img/sso-azure-step-05.png)
+
+### Step 6: Upload SAML-based Sign-on metadata file
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-azure.mdx:59]:
```suggestion
## Step 6: Upload SAML-based Sign-on metadata file [#upload-saml-metadata]
```

File: apps/docs/pages/guides/platform/sso/sso-with-azure.mdx [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+**Make sure you input these additional settings.**
+
+| Setting | Value |
+| ------- | ----- |
+| Sign on URL | `https://app.supabase.com/sign-in-sso` |
+| Relay State | `https://app.supabase.com` |
+
+Finally, click the *Save* button to save the configuration.
+
+### Step 7: Obtain metadata URL and send to Supabase
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-azure.mdx:89]:
```suggestion
## Step 7: Obtain metadata URL and send to Supabase [#send-metadata-url]
```

File: apps/docs/pages/guides/platform/sso/sso-with-azure.mdx [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+contact and await further instructions. If you're not clear who to send this
+link to or need further assistance, please reach out to
+[support@supabase.com](mailto:support@supabase.com).
+
+**Do not test the login until you have heard back from the support contact.**
+
+![Azure AD console: Supabase application, SAML Certificates card
+shown, App Federation Metadata Url highlighted](/docs/img/sso-azure-step-07.png)
+
+### Step 8: Wait for confirmation
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-azure.mdx:103]:
```suggestion
## Step 8: Wait for confirmation [#confirmation]
```

File: apps/docs/pages/guides/platform/sso/sso-with-azure.mdx [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+
+Please wait for confirmation or further instructions from your support contact
+at Supabase before proceeding to the next step. It usually takes us 1 business
+day to configure SSO for you.
+
+### Step 9: Test single sign-on
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-azure.mdx:109]:
```suggestion
## Step 9: Test single sign-on [#testing]
```

File: apps/docs/pages/guides/platform/sso/sso-with-azure.mdx [hunk 10]
```diff
@@ -0,0 +1,127 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+import { Button, IconArrowDown } from 'ui'
+
+export const meta = {
+  id: 'sso-with-azure',
+  title: 'Setup SSO with Azure AD',
+  description: 'Configuration instructions for single sign-on with Azure AD (Entra)',
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-azure.mdx:7]:
```suggestion
  title: 'Set Up SSO with Azure AD',
  description: 'Configure single sign-on with Azure AD (Microsoft Entra).',
```

File: apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx [hunk 11]
```diff
@@ -0,0 +1,127 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'sso-with-gsuite',
+  title: 'Setup SSO with Google Workspace',
+  description: 'Configuration instructions for single sign-on with Google Workspace (GSuite)',
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx:6]:
```suggestion
  title: 'Set Up SSO with Google Workspace',
  description: 'Configure single sign-on with Google Workspace (GSuite).',
```

File: apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+}
+
+Supabase supports single sign-on (SSO) using Google Workspace (formerly known
+as GSuite).
+
+## Step-by-step guide
+
+### Step 1: Open the Google Workspace Web and mobile apps console
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx:14]:
```suggestion
## Step 1: Open the Google Workspace Web and mobile apps console [#google-workspace-console]
```

File: apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx [hunk 13]
```diff
@@ -0,0 +1,127 @@
...
+
+![Google Workspace: Web and mobile apps admin
+console](/docs/img/sso-gsuite-step-01.png)
+
+### Step 2: Choose Add custom SAML app
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx:19]:
```suggestion
## Step 2: Choose Add custom SAML app [#add-custom-saml-app]
```

File: apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx [hunk 14]
```diff
@@ -0,0 +1,127 @@
...
+
+From the *Add app* button in the toolbar choose *Add custom SAML app*.
+
+![Google Workspace: Web and mobile apps admin console, Add custom SAML app
+selected](/docs/img/sso-gsuite-step-02.png)
+
+### Step 3: Fill out App details
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx:26]:
```suggestion
## Step 3: Fill out app details [#add-app-details]
```

File: apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx [hunk 15]
```diff
@@ -0,0 +1,127 @@
...
+
+The information you enter here is for visibility into your Google Workspace.
+You can choose any values you like. `Supabase` as a name works well for most
+use cases. Optionally enter a description.
+
+![Google Workspace: Web and mobile apps admin console, Add custom SAML, App
+details screen](/docs/img/sso-gsuite-step-03.png)
+
+### Step 4: Download IdP metadata
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx:35]:
```suggestion
## Step 4: Download IdP metadata [#download-idp-metadata]
```

File: apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx [hunk 16]
```diff
@@ -0,0 +1,127 @@
...
+
+It's very important to Send this file to your support contact at Supabase to
+complete the SSO setup process. If you're not sure where to send this file, you
+can always reach us at [support@supabase.com](mailto:support@supabase.com).
+
+**Important: Make sure the certificate as shown on screen has at least 1 year
+before it expires. Mark down this date in your calendar so you will be reminded
+that you need to update the certificate without any downtime for your users.**
+
+### Step 5: Add service provider details
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx:51]:
```suggestion
## Step 5: Add service provider details [#add-service-provider-details]
```

File: apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx [hunk 17]
```diff
@@ -0,0 +1,127 @@
...
+| ACS URL | `https://alt.supabase.io/auth/v1/sso/saml/acs` |
+| Entity ID | `https://alt.supabase.io/auth/v1/sso/saml/metadata` |
+| Start URL | `https://app.supabase.com` |
+| Name ID format | PERSISTENT |
+| Name ID | *Basic Information > Primary email* |
+
+![Google Workspace: Web and mobile apps admin console, Add custom SAML,
+Service provider details screen](/docs/img/sso-gsuite-step-05.png)
+
+### Step 6: Configure Attribute mapping
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx:66]:
```suggestion
## Step 6: Configure Attribute mapping [#configure-attribute-mapping]
```

File: apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx [hunk 18]
```diff
@@ -0,0 +1,127 @@
...
+shown below are optional and configurable depending on your Google Workspace
+setup. If in doubt, replicate the same config as shown.
+
+Please share any changes, if any, from this screen with your Supabase support
+contact.
+
+![Google Workspace: Web and mobile apps admin console, Add custom SAML,
+Attribute mapping](/docs/img/sso-gsuite-step-06.png)
+
+### Step 7: Wait for confirmation
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx:81]:
```suggestion
## Step 7: Wait for confirmation [#confirmation]
```

File: apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx [hunk 19]
```diff
@@ -0,0 +1,127 @@
...
+
+This information needs to be entered into Supabase before SSO is activated
+end-to-end.
+
+Wait for confirmation that this information has successfully been added to
+Supabase. It usually takes us 1 business day to configure this information
+for you.
+Supabase.
+
+### Step 8: Configure user access
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx:97]:
```suggestion
## Step 8: Configure user access  [#configure-user-access]
```

File: apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx [hunk 20]
```diff
@@ -0,0 +1,127 @@
...
+You can configure this access by clicking on the *User
+access* card (or down-arrow). Follow the instructions on screen.
+
+Changes from this step sometimes take a while to propagate across Google’s
+systems. Please wait at least 15 minutes before proceeding to the next step.
+
+![Google Workspace: Web and mobile apps admin console, Supabase app
+screen](/docs/img/sso-gsuite-step-08.png)
+
+### Step 9: Test single sign-on
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx:112]:
```suggestion
## Step 9: Test single sign-on [#testing]
```

File: apps/docs/pages/guides/platform/sso/sso-with-azure.mdx [hunk 21]
```diff
@@ -0,0 +1,127 @@
...
+
+Please wait for confirmation or further instructions from your support contact
+at Supabase before proceeding to the next step. It usually takes us 1 business
+day to configure SSO for you.
+
+### Step 9: Test single sign-on
+
+*Testing sign-on before your Azure AD has been registered with Supabase will
+not work. Make sure you've received confirmation from your support contact at
+Supabase as laid out in [Step 8](#step-8-wait-for-confirmation).*
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-azure.mdx:113]:
```suggestion
Supabase as laid out in the [confirmation](#confirmation) step.*
```

File: apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx [hunk 22]
```diff
@@ -0,0 +1,127 @@
...
+
+![Google Workspace: Web and mobile apps admin console, Add custom SAML,
+Attribute mapping](/docs/img/sso-gsuite-step-06.png)
+
+### Step 7: Wait for confirmation
+
+Once you’ve configured the Google Workspace app as shown above, make sure you
+send the metadata file you downloaded from [Step
+4](#step-4-download-idp-metadata) and information regarding the
+attribute mapping from [Step 6](#step-6-configure-attribute-mappings) (if any
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-gsuite.mdx:86]:
```suggestion
send the [metadata file you downloaded](#download-idp-metadata)
and information regarding the [attribute mapping](#configure-attribute-mappings) (if any
```

File: apps/docs/pages/guides/platform/sso/sso-with-okta.mdx [hunk 23]
```diff
@@ -0,0 +1,116 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'sso-with-okta',
+  title: 'Setup SSO with Okta',
+  description: 'Configuration instructions for single sign-on with Okta',
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-okta.mdx:6]:
```suggestion
  title: 'Set Up SSO with Okta',
  description: 'Configure single sign-on with Okta.',
```

File: apps/docs/pages/guides/platform/sso/sso-with-okta.mdx [hunk 24]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+Supabase supports single sign-on (SSO) using Okta.
+
+## Step-by-step guide
+
+### Step 1: Choose Create App Integration in the Applications dashboard
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-okta.mdx:13]:
```suggestion
## Step 1: Choose Create App Integration in the Applications dashboard [#create-app-integration]
```

File: apps/docs/pages/guides/platform/sso/sso-with-okta.mdx [hunk 25]
```diff
@@ -0,0 +1,116 @@
...
+
+Navigate to the Applications dashboard of the Okta admin console. Choose the
+*Create App Integration* button from the toolbar.
+
+![Okta dashboard: Create App Integration
+button](/docs/img/sso-okta-step-01.png)
+
+### Step 2: Choose SAML 2.0 in the app integration dialog
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-okta.mdx:21]:
```suggestion
## Step 2: Choose SAML 2.0 in the app integration dialog [#create-saml-app]
```

File: apps/docs/pages/guides/platform/sso/sso-with-okta.mdx [hunk 26]
```diff
@@ -0,0 +1,116 @@
...
+
+Supabase supports the SAML 2.0 SSO protocol. Choose it from the *Create a new
+app integration* dialog.
+
+![Okta dashboard: Create new app integration dialog](/docs/img/sso-okta-step-02.png)
+
+### Step 3: Fill out General Settings
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-okta.mdx:28]:
```suggestion
## Step 3: Fill out General Settings [#add-general-settings]
```

File: apps/docs/pages/guides/platform/sso/sso-with-okta.mdx [hunk 27]
```diff
@@ -0,0 +1,116 @@
...
+
+The information you enter here is for visibility into your Okta applications
+menu.  You can choose any values you like. `Supabase` as a name works well for
+most use cases.
+
+![Okta dashboard: Create SAML Integration
+wizard](/docs/img/sso-okta-step-03.png)
+
+### Step 4: Fill out SAML Settings
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-okta.mdx:37]:
```suggestion
## Step 4: Fill out SAML Settings [#add-saml-settings]
```

File: apps/docs/pages/guides/platform/sso/sso-with-okta.mdx [hunk 28]
```diff
@@ -0,0 +1,116 @@
...
+| Audience URI (SP Entity ID) | `https://app.supabase.com/auth/v1/sso/saml/metadata` |
+| Default RelayState | `https://app.supabase.com` |
+| Name ID format | EmailAddress |
+| Application username | Email |
+| Update application username on | Create and update |
+
+![Okta dashboard: Create SAML Integration
+wizard, Configure SAML step](/docs/img/sso-okta-step-04.png)
+
+### Step 5: Fill out Attribute Statements
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-okta.mdx:56]:
```suggestion
## Step 5: Fill out Attribute Statements [#add-attribute-statements]
```

File: apps/docs/pages/guides/platform/sso/sso-with-okta.mdx [hunk 29]
```diff
@@ -0,0 +1,116 @@
...
+shown below are optional and configurable depending on your Okta
+setup. If in doubt, replicate the same config as shown.
+
+Please share any changes, if any, from this screen with your Supabase support
+contact.
+
+![Okta dashboard: Attribute Statements configuration
+screen](/docs/img/sso-okta-step-05.png)
+
+### Step 6: Obtain IdP metadata URL
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-okta.mdx:70]:
```suggestion
## Step 6: Obtain IdP metadata URL [#idp-metadata-url]
```

File: apps/docs/pages/guides/platform/sso/sso-with-okta.mdx [hunk 30]
```diff
@@ -0,0 +1,116 @@
...
+Copy this URL and send it to your support contact and await further
+instructions. If you're not clear who to send this link to or need further
+assistance, please reach out to
+[support@supabase.com](mailto:support@supabase.com).
+
+The link usually has this structure: `https://<okta-org>.okta.com/apps/<app-id>/sso/saml/metadata`
+
+![Okta dashboard: SAML Signing Certificates, Actions button highlighted](/docs/img/sso-okta-step-06.png)
+
+### Step 7: Wait for confirmation
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-okta.mdx:90]:
```suggestion
## Step 7: Wait for confirmation [#confirmation]
```

File: apps/docs/pages/guides/platform/sso/sso-with-okta.mdx [hunk 31]
```diff
@@ -0,0 +1,116 @@
...
+metadata URL from [Step 6](#step-6-obtain-idp-metadata-url) and information
+regarding the attribute statements from [Step
+5](#step-5-fill-out-attribute-statements) (if any changes are applicable) to
+your support contact at Supabase.
+
+Wait for confirmation that this information has successfully been added to
+Supabase. It usually takes us 1 business day to configure this information
+for you.
+
+### Step 8: Test single sign-on
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-okta.mdx:102]:
```suggestion
## Step 8: Test single sign-on [#testing]
```

File: apps/docs/pages/guides/platform/sso/sso-with-okta.mdx [hunk 32]
```diff
@@ -0,0 +1,116 @@
...
+The link usually has this structure: `https://<okta-org>.okta.com/apps/<app-id>/sso/saml/metadata`
+
+![Okta dashboard: SAML Signing Certificates, Actions button highlighted](/docs/img/sso-okta-step-06.png)
+
+### Step 7: Wait for confirmation
+
+Once you’ve configured the Okta app as shown above, make sure you send the
+metadata URL from [Step 6](#step-6-obtain-idp-metadata-url) and information
+regarding the attribute statements from [Step
+5](#step-5-fill-out-attribute-statements) (if any changes are applicable) to
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-okta.mdx:95]:
```suggestion
[metadata URL](idp-metadata-url) and information regarding the
[attribute statements](#add-attribute-statements) (if any changes are applicable) to
```

**Review by dannykng — APPROVED** [review]
Left a bunch of suggestions but it's mostly high-level feedback. 😅

File: apps/docs/pages/guides/platform/sso/sso-with-okta.mdx [hunk 33]
```diff
@@ -0,0 +1,115 @@
...
+[support@supabase.com](mailto:support@supabase.com).
+
+The link usually has this structure: `https://<okta-org>.okta.com/apps/<app-id>/sso/saml/metadata`
+
+![Okta dashboard: SAML Signing Certificates, Actions button highlighted](/docs/img/sso-okta-step-06.png)
+
+## Step 7: Wait for confirmation [#confirmation]
+
+Once you’ve configured the Okta app as shown above, make sure you send the
+[metadata URL](idp-metadata-url) and information regarding the
```
Comment (dannykng) [inline apps/docs/pages/guides/platform/sso/sso-with-okta.mdx:93]: I forgot the `#` here 😅 
```suggestion
[metadata URL](#idp-metadata-url) and information regarding the
```

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 34]
```diff
@@ -486,6 +486,24 @@ export const platform = {
...
+          url: '/guides/platform/sso/enable-sso',
+          items: [],
+        },
+        { name: 'Setup SSO with Azure AD', url: '/guides/platform/sso/sso-with-azure', items: [] },
+        {
+          name: 'Setup SSO with Google Workspaces',
+          url: '/guides/platform/sso/sso-with-gsuite',
+          items: [],
+        },
+        { name: 'Setup SSO with Okta', url: '/guides/platform/sso/sso-with-okta', items: [] },
```
[the remark this answers is not in the document]
Comment (dannykng) [reply apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts]: @kangmingtay sorry should have mentioned that doing this necessitates renaming the files (and moving `enable-sso.mdx` up one level in the directory).

[the remark this answers is not in the document]
Comment (kangmingtay) [reply apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts]: yup i'm working on that

**Review by MildTomato — APPROVED** [review]
get em in.

I'll add next/image support in today so your images don't take forever

**Comment by kangmingtay** [comment]
@MildTomato doing a last round of proofreading before i merge

**Review by kangmingtay — APPROVED** [review]

**Pull request merged**